share/spice/people_in_space/people_in_space.css

```diff
@@ -1,15 +1,98 @@
-.tile--people_in_space .tile__icon {
-    width: 20px;
-    height: 20px;
-    padding: 6px;
-    border-radius: 2px;
-    background-color: #e8e8e8;
+/* tiles */
+.zci--people_in_space .tile__media {
+    height: 100px;
+    width: 100px;
+    border-radius: 100%;
+    margin-bottom: 0.5em;
 }
 
-.tile--people_in_space .tile__body {
-    height: 10em;
+.zci--people_in_space .tile--people_in_space {
+    height: 205px;
+    margin-bottom: 10px;
+    width: 150px;
+    padding: 20px;
 }
 
-.zci--people_in_space .text--primary {
-    font-size: 1.5em;
+.zci--people_in_space .name {
+    font-size: 1.277em;
+    line-height: 1.2;
 }
+
+.zci--people_in_space .info {
+	text-align: center;
+    position: inherit;
+    display: table-cell; 
+    vertical-align: middle;
+    height: 55px;
+    bottom: 0px;
+    width: 100%;
+}
+
+.zci--people_in_space .days {
+    text-align: center;
+    display:inline-block;
+    clear: left;
+    margin-top: 3px;
+}
+
+.zci--people_in_space .tile__body {
+	padding: 0;
+	padding-top: 0;
+}
+
+.zci--people_in_space .image-under img {
+   	height: 130px;
+    width: 100px;
+}
+
+.zci--people_in_space .image-over img {
+    border: 3px solid #fff;
+    height: 32px;
+    width: 32px;
+	border-radius: 100%;
+}
+
+.zci--people_in_space .image-under {
+    position: absolute;
+}
+
+.zci--people_in_space .image-over {
+    position: absolute;
+    z-index: 5;
+    top: 86px;
+    left: 95px;
+}
+
+/* detail pane */
+.zci--people_in_space .details .right {
+    margin-left: 2em
+}
+
+.zci--people_in_space .details {
+    line-height: 1.3em;
+    width: 50em;
+}
+
+.zci--people_in_space .details > ul {
+    float:left;
+}
+
+.zci--people_in_space .callout {
+    clear: left;
+    margin-top: 0.5em;
+    line-height: 1.3em;
+}
+
+.zci--people_in_space .zci__img {
+    max-height: 100%;
+    max-width: 100%;
```

Comment:
I have a hunch that you need to also set `width: 100%` and `height: 100%` for IE8